src/sysupdate/sysupdate-resource.c

```diff
@@ -648,7 +649,11 @@ int resource_resolve_path(
                 _cleanup_free_ char *resolved = NULL, *relative_to = NULL;
                 ChaseFlags chase_flags = CHASE_PREFIX_ROOT;
 
-                if (rr->path_relative_to == PATH_RELATIVE_TO_ROOT) {
+                if (rr->path_relative_to == PATH_RELATIVE_TO_DIRECTORY) {
+                        relative_to = strdup(empty_to_root(relative_to_directory));
```

Comment:
hmm, i wonder if this is the right approach with this.

i see two other options here:

1. let's make the tool fail if "directory" is specified but none given.
2. default to the caller's cwd?

I think failing would be best?